tools/clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -121,7 +121,7 @@ void CGDebugInfo::setLocation(SourceLocation Loc) {
 
   SourceManager &SM = CGM.getContext().getSourceManager();
   auto *Scope = cast<llvm::DIScope>(LexicalBlockStack.back());
-  PresumedLoc PCLoc = SM.getPresumedLoc(CurLoc);
+  PresumedLoc PCLoc = SM.getPresumedLoc(CurLoc, false); // HLSL Change
```

Comment:
I know Tex is a fan of adding /* UseLineDirectives */ comments to parameter literals. I can see how that would help here.